Doc/library/__main__.rst

```diff
@@ -251,9 +251,10 @@ attribute will include the package's path if imported::
     >>> asyncio.__main__.__name__
     'asyncio.__main__'
 
-This won't work for ``__main__.py`` files in the root directory of a .zip file
-though.  Hence, for consistency, minimal ``__main__.py`` like the :mod:`venv`
-one mentioned below are preferred.
+
```

Comment:
There appears to be an extra blank line inserted above the paragraph — I don't know that it's a _problem_ (or even if it will affect the output), but it certainly seems unnecessary.
```suggestion
```